tests/tests/test-proxy-governance.ts

```diff
@@ -0,0 +1,66 @@
+import { expect } from "chai";
+import Keyring from "@polkadot/keyring";
+import {
+  ALITH_PRIV_KEY,
+  DOROTHY,
+  DOROTHY_PRIV_KEY,
+  ETHAN,
+  ETHAN_PRIVKEY,
+  GENESIS_ACCOUNT_BALANCE,
+  PROPOSAL_AMOUNT,
+  VOTE_AMOUNT,
+} from "../util/constants";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+
+describeDevMoonbeam("Proxing governance", (context) => {
+  it("should be able to vote on behalf of the delegate account", async function () {
+    this.timeout(40_000);
```

Comment:
```suggestion
    // Because we have to produce a lot (3600 at time or writing) of blocks to exhaust the LaunchPeriod
    this.timeout(40_000);
```